tests/acceptance/generate-component.spec.js

```diff
@@ -178,6 +178,20 @@ describe('Acceptance: ng generate component', function () {
       });
   });
 
+  it(`nonExistingDir${path.sep}myComp will create dir and succeed`, () => {
+    const testPath =
+      path.join(root, 'tmp', 'foo', 'src', 'app', 'non-existing-dir', 'my-comp', 'my-comp.component.ts');
+    const appModule = path.join(root, 'tmp', 'foo', 'src', 'app', 'app.module.ts');
+    return ng(['generate', 'component', `non-existing-dir${path.sep}myComp`])
```

Comment:
Can you test `non/existing/dir/myComp` instead? It should be fine to use `/` everywhere, and I'd also like to be sure that nested dirs work.